src/components/attendee/BasicInfo/ProfileModalAvatar/ProfileModalAvatar.module.scss

```diff
@@ -1,22 +1,30 @@
+@use "scss/attendee/space";
+@use "scss/attendee/theme";
+@use "scss/attendee/font";
+@use "scss/attendee/border";
+
 .profileModalAvatar {
-  width: 84px;
-  height: 84px;
+  width: space.empty(7);
```

Comment:
[NOTE] Maybe we should have few icon sizes put into a list of constants somewhere.

Reply:
added to constants [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)